src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -526,9 +518,7 @@ const EnvironmentPicker = ({
                 <div className="cb-environment-picker-container">
                     <FontIcon iconName={'Database'} />
                     <span className="cb-environment-picker-container-title">
-                        {getContainerText(
-                            state.containerItems.selectedContainer
-                        )}
+                        {containerDisplayText}
                     </span>
                 </div>
             )}
```

Comment:
you still need to fix this accessibility issue with the dialog being in the DOM when not visible.